CHANGELOG.md

```diff
@@ -2,12 +2,48 @@
 
 ## Unreleased
 
+## v0.82.0
+
 ### 🛑 Breaking changes 🛑
 
 - (Splunk) Fluentd installation ***disabled*** by default for the Linux and Windows installer scripts ([#3369](https://github.com/signalfx/splunk-otel-collector/pull/3369))
```

Comment:
QQ: What about Ansible, Puppet and Salt releases? When are those changes expected to be merged?

Reply:
Chef: https://github.com/signalfx/splunk-otel-collector/pull/3442
Ansible: https://github.com/signalfx/splunk-otel-collector/pull/3440
We can review those 2 and look at merging them ahead of the release.

Reply:
They are released as different installation artifacts and not related to this release

Reply:
True, these are released separately.

Reply:
Yes, not related to this one. Since customers use different artifacts wanted to get up to speed with the ones this change has been made to and the ones remaining for field comms.